docs/sphinx/conf.py.in

```diff
@@ -370,5 +370,6 @@ linkcheck_ignore = ['https://imspector.mpibpc.mpg.de',
     'https://code.google.com/archive/p/jj2000/',
     'https://support.apple.com/downloads/quicktime',
     'https://support.apple.com/quicktime',
-    'https://www.micro-manager.org'
+    'https://www.micro-manager.org',
+    'https://nifti.nimh.nih.gov/nifti-1/*'
```

Comment:
Shouldn't need the trailing `*`?

Reply:
We do have a couple of different links, https://nifti.nimh.nih.gov/nifti-1 and https://nifti.nimh.nih.gov/nifti-1/data for example.

Reply:
I wasn't sure whether it was needed or not but this construction seemed to work fine so I left it as is. Happy to remove if that's the consensus though

Reply:
A link to https://nifti.nimh.nih.gov/nifti-1 or a link to https://nifti.nimh.nih.gov/nifti-1/? I see only the latter, at least on the NIfTI page.

These are *partial*-line regexp matches, they don't follow the syntax of, say, `find`'s `-name` argument. Leaving this PR as-is is confusing because what it actually means to a Python regexp is not what it looks like the author was thinking. I'd definitely suggest removing the trailing `*`. *If* we need to match `https://nifti.nimh.nih.gov/nifti-1` too then easiest (but imperfect) would be to remove the then-trailing `/` too but so far I don't see that we do.

Reply:
Removed, thanks for the explanation @mtbc

Reply:
Thank you. If the build's no longer green then you can make me fix it. :smiley: